Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/Payment Flow/PaymentFormViewController.m

```diff
@@ -62,8 +62,8 @@ - (void)pay {
                     break;
                 }
                 case STPPaymentHandlerActionStatusSucceeded: {
-                    UIAlertController *alert = [Utils createAlertControllerWithTitle:@"Payment succeeded." andMessage:error.localizedDescription okCompletion:nil cancelCompletion:nil];
-                    [self presentViewController:alert animated:YES completion:nil];
+                    [self performSegueWithIdentifier:@"PaymentConfirmedSegue" sender:nil];
+                    self.tabBarController.hidesBottomBarWhenPushed = YES;
```

Comment:
When do you set this back to `NO`?

Reply:
"If YES, the bottom bar remains hidden until the view controller is popped from the stack." 
This takes the user to the final confirmation page (screenshotted below), and when the user clicks "Finish", the navController pops back to the root view controller. 
<img width="375" alt="Screen Shot 2020-07-29 at 10 26 48 AM" src="https://user-images.githubusercontent.com/27475332/88832778-254f5380-d186-11ea-94e8-5728871857fb.png">

Reply:
I think when the view controller is popped back the tab bar will be visible again because the tab bar view controller is at the top of the view stack, but then if you push another view controller the tab bar will be hidden again because `hidesBottomBarWhenPushed` is still `YES`. I might be wrong though, but you should try it to be safe.

Reply:
Tested it a couple times, looks like it should work the way it should right now! Thanks for catching this though